logprep/event_generator/kafka/run_load_tester.py

```diff
@@ -0,0 +1,29 @@
+"""Main module for the load-tester"""
+from multiprocessing import Manager
+from pathlib import Path
+
+from logprep.event_generator.kafka.configuration import load_config
+from logprep.event_generator.kafka.logger import create_logger
+from logprep.event_generator.kafka.process_runner import run_processes
+from logprep.event_generator.kafka.util import print_results, print_startup_info
+
+
+class LoadTester:
+    """Load Tester to generate events from kafka to kafka."""
+
+    def __init__(self, config, file):
+        self.config = Path(config)
+        self.file = file
+
+    def run(self):
+        """Start function for the load-tester"""
+        config = load_config(self.config, self.file)
```

Comment:
I just turned the cli `--file` flag into an actual path. Can you please try it out again?

Reply:
Thank you it works.
I did notice however that "--file" only works if a connection to Kafka exists, even if the documents are not retrieved from Kafka.
I believe that's a problem my original commit already had.
Maybe this could be changed in the future.